docs/source-maps.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[source-maps]]
+== Source Map Support
+
+The Elastic APM Node.js agent supports source maps by default.
+If your transpile your source code and supply a source map,
+The agent will be able to collect the correct stack traces and even the original source code if available.
+
+To take advantage of this, simply make sure that your transpiled source code contains a `sourceMappingURL` comment at the bottom of each JavaScript file.
+It can either point to a source map file on disk:
+
+[source,js]
+----
+//# sourceMappingURL=/path/to/file.js.map
+----
+
+Or you can inline the source map using base64 encoding:
+
+[source,js]
+----
+//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiIiwic291cmNlcyI6WyJmb28uanMiLCJiYXIuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7Ozs7O1VBQ0c7Ozs7Ozs7Ozs7Ozs7O3NCQ0RIO3NCQUNBIn0=
+----
+
+All modern build-toolchains support generating source maps and adding these comments to the transpiled source code.
+
+[[original-source-code]]
+=== Original source code
+
+Elastic APM uses source maps for two things:
```

Comment:
Change to: Elastic APM uses source maps for two purposes: to collect stack traces that point to your original source code,  _and_ to collect the original source code as inline code snippets related to each frame in your stack traces.